addons/explosives/CfgAmmo.hpp

```diff
@@ -67,7 +65,7 @@ class CfgAmmo {
         GVAR(size) = 0;
     };
 
-    // More sensitive pressure-fuze for the Vanilla AT Mine.
+    // More sensitive pressure-fuse for the Vanilla AT Mine
```

Comment:
It really doesn't matter but I think fuze is correct here
it's something like:
- fuse - like the classic burning/sparking rope on a stick of dynamite
- fuze - something more complicated like a sensor or timer that triggers an explosive

Reply:
It's AE vs BE shenanigans, but seeing as we have `fuse` everywhere else, I figured I would bring it in line, despite being BE.